requirements.txt

```diff
@@ -22,5 +22,8 @@
 mkdocs>=1.1
 Pygments>=2.4
 markdown>=3.2
-pymdown-extensions>=7.0
+# pymdown-extensions>=7.0
 mkdocs-material-extensions>=1.0
+
+# Use feature branch temporarily
+git+https://github.com/facelessuser/pymdown-extensions.git@feature/tabbed-alternate
```

Comment:
Can you change this to?

```
pymdown-extensions @ git+https://github.com/facelessuser/pymdown-extensions.git@feature/tabbed-alternate
```

Reply:
Having trouble installing with `pip`

Reply:
Fixed, I wasn't aware of that syntax.

Reply:
Yeah, it's new-ish. I ran into it with a project at my work.

Reply:
Setuptools doesn't like it in the old way.